src/main/java/seedu/address/model/person/RemindPredicate.java

```diff
@@ -24,8 +22,7 @@ public RemindPredicate(int days) {
 
     @Override
     public boolean test(Person person) {
-        long dueDays = ChronoUnit.DAYS.between(LocalDate.now(), person.getPolicy().getPolicyExpiryDate().date);
-        return dueDays >= VALID_DAYS_STARTS_FROM && dueDays <= days;
+        return ChronoUnit.DAYS.between(LocalDate.now(), person.getPolicy().getPolicyExpiryDate().date) <= days;
```

Comment:
Do we need to separately handle cases:
i) today date is before policy expiry date
ii) today date is after policy expiry date

Reply:
I agree, let me work on that. Thanks!